### PR TITLE
Update Pyqt 5.4.2  and SIP 4.16.8

### PR DESCRIPTION
**Pull request by noplay**
Upgrade to the last version of PyQT and his dependency SIP.

### CODE REVIEW

File: Library/Formula/sip.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Sip < Formula
   desc "Tool to create Python bindings for C and C++ libraries"
   homepage "http://www.riverbankcomputing.co.uk/software/sip"
-  url "https://downloads.sf.net/project/pyqt/sip/sip-4.16.5/sip-4.16.5.tar.gz"
-  sha1 "d5d7b6765de8634eccf48a250dbd915f01b2a771"
+  url "https://downloads.sf.net/project/pyqt/sip/sip-4.16.8/sip-4.16.8.tar.gz"
+  sha1 "0dd2463e5e1fc564c399889fc72baa67c35fcaab"
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/sip.rb]: Remove this `revision` line, thanks.

File: Library/Formula/pyqt5.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 class Pyqt5 < Formula
   desc "Python bindings for v5 of Qt"
   homepage "http://www.riverbankcomputing.co.uk/software/pyqt/download5"
-  url "https://downloads.sf.net/project/pyqt/PyQt5/PyQt-5.4/PyQt-gpl-5.4.tar.gz"
-  sha1 "057e6b32c43e673e79f876fb9b6f33d3072edfc2"
+  url "https://downloads.sf.net/project/pyqt/PyQt5/PyQt-5.4.2/PyQt-gpl-5.4.2.tar.gz"
+  sha1 "ceb5f6e088923beaa9885226a3e677fec089964e"
```
Comment (mikemcquaid) [inline Library/Formula/pyqt5.rb]: Mind making this `sha256` instead? Thanks!

File: Library/Formula/sip.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class Sip < Formula
   desc "Tool to create Python bindings for C and C++ libraries"
   homepage "http://www.riverbankcomputing.co.uk/software/sip"
-  url "https://downloads.sf.net/project/pyqt/sip/sip-4.16.5/sip-4.16.5.tar.gz"
-  sha1 "d5d7b6765de8634eccf48a250dbd915f01b2a771"
+  url "https://downloads.sf.net/project/pyqt/sip/sip-4.16.8/sip-4.16.8.tar.gz"
+  sha1 "0dd2463e5e1fc564c399889fc72baa67c35fcaab"
```
Comment (mikemcquaid) [inline Library/Formula/sip.rb]: Mind making this `sha256` instead? Thanks!

**Comment by noplay** [comment]
I drop the revision and change sha1 to sha256

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/7c690955e242c73743057e5adf2d13be7c584ba7 and https://github.com/Homebrew/homebrew/commit/423d31aa4599abc884b771b99a5933d59ab759a9. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Please note for future the preferred commit style for version bumps is a simple `pyqt5 5.4.2` or similar, Thanks!

**Pull request closed without merging**